app/src/main/java/de/danoeh/antennapod/adapter/DefaultActionButtonCallback.java

```diff
@@ -34,17 +45,26 @@ public DefaultActionButtonCallback(Context context) {
     @Override
     public void onActionButtonPressed(final FeedItem item) {
 
-
         if (item.hasMedia()) {
             final FeedMedia media = item.getMedia();
             boolean isDownloading = DownloadRequester.getInstance().isDownloadingFile(media);
             if (!isDownloading && !media.isDownloaded()) {
-                try {
-                    DBTasks.downloadFeedItems(context, item);
-                    Toast.makeText(context, R.string.status_downloading_label, Toast.LENGTH_SHORT).show();
-                } catch (DownloadRequestException e) {
-                    e.printStackTrace();
-                    DownloadRequestErrorDialogCreator.newRequestErrorDialog(context, e.getMessage());
+                if (UserPreferences.isAllowMobileUpdate() || NetworkUtils.connectedToWifi(context) ||
```

Comment:
Should we move this conditional in to NetworkUtils?

``` UserPreferences.isAllowMobileUpdate() || NetworkUtils.connectedToWifi(context) ``` -> ```NetworkUtils.isDownloadAllowed()```?

I know we check these two things elsewhere.  Not that important. If it's only used in this spot maybe just keep it.